src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -120,104 +118,87 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
 
   const renderExtension = () => (
     showExtension && (
-      <Box width={0.25}>
-        <TextField
-          autoCompleteType={disableAutocomplete ? 'off' : 'tel-extension'}
-          id="phoneExtension"
-          isDisabled={loading}
-          label={extensionLabel ?? ''}
-          name="extension"
-          onChange={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
-            setExtension(e.currentTarget.value);
-          }}
-          testId="extension"
-          value={extension}
-        />
-      </Box>
+      <TextField
+        autoCompleteType={disableAutocomplete ? 'off' : 'tel-extension'}
+        id="phoneExtension"
+        isDisabled={loading}
+        isOptional
+        label={extensionLabel ?? ''}
+        name="extension"
+        onChange={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+          setExtension(e.currentTarget.value);
+        }}
+        testId="extension"
+        value={extension}
+      />
     )
   );
 
   const renderCountrySelect = () => (
-    <Box marginBlockEnd={4}>
```

Comment:
When talking to the design team, it was determined that we don't need this additional margin and now the inputs are spaced consistently.